src/main/webapp/feed.html

```diff
@@ -40,6 +49,84 @@ <h1>
           })
         }
       })
+
+      let tagSelect = document.getElementById("tag-select")
```

Comment:
Kind of a nitpick but I think this block of code and the block above it would be a little less scary if they were wrapped up in a window.onload = function () {} or something like that--there shouldn't be any problems but just in case (and for consistency's sake) we should probably make sure that the HTML elements r loaded.

Reply:
Agreed!